### PR TITLE
Milestone:2 Microverse project HTML Forms

### PR DESCRIPTION
**Pull request by talhawaqar**
Create mint.com signup form replica according to the requirements of Microverse

### CODE REVIEW

**Comment by addod19** [comment]
## Pull  Request under review by Daniel Larbi Addo (TSE)

File: index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+          <form>
+            <div class="input-fields">
+              <label for="email">Email</label>
+              <input type="email" name="email" id="email" />
+            </div>
+
+            <div class="input-fields">
+              <label for="phone">Phone (recommended)</label>
+              <input type="number" name="phone" id="phone" />
+              <p class="phone-note">Standard call, messaging or data rates may apply.</p>
```
Comment (addod19) [inline index.html]: Let us improve semantics by using ```<main>``` instead of ```<div class="main-form">```

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @talhawaqar and @visva-dev, congratulations on a great job done :+1:  Let us improve our code and make it even better by fixing the comments below:
- Kindly enable stickler for the repo by visiting stickler-ci.com
- Kindly change the type of the input to ```tel``` ``` <input type="number" name="phone" id="phone" />```
- Currently, the page is missing a ```<footer>``` let us include the ```<footer>```
- Let us apply the DRY rule to our CSS file, kindly read [this](https://speckyboy.com/good-bad-css-practices/) for more insight
- Kindly add emails and twitter handles to our README file

Kindly make these changes and submit another code review :smile: 
_Happy coding_ :man_technologist: 
Daniel Larbi Addo on slack

**Comment by mikenath223** [comment]
### Pull request currently under review by Michgolden Ukeje (TSE).
Kindly exercise patience 👋

**Review by mikenath223 — APPROVED** [review]
### Hi Team 👋

You both have done quite well on this project @talhawaqar and @visva-dev  👍

This project meets the requirements for approval.

**STATUS: APPROVED ✅**


Don't forget to fill the project completion form👍
Good luck with the next project.

HAPPY CODING👨‍💻

**Pull request merged**